.github/workflows/test.yml

```diff
@@ -38,18 +41,32 @@ jobs:
       with:
         version: ${{ matrix.emacs-version }}
 
-    - uses: emacs-eask/setup-eask@master
+    - name: Setup Eask
+      uses: emacs-eask/setup-eask@master
+      if: matrix.os == 'ubuntu-latest' || matrix.os == 'macos-latest'
```

Comment:
I think `windows-latest` is missing here? 🤔

Reply:
you're right, good catch, thanks